test/checks/design/InterfaceIsTypeCheckTest.hx

```diff
@@ -0,0 +1,40 @@
+package checks.design;
+
+import checkstyle.checks.design.InterfaceIsTypeCheck;
+
+class InterfaceIsTypeCheckTest extends CheckTestCase<InterfaceIsTypeCheckTests> {
+
+	public function testWithJustVars() {
+		assertMsg(new InterfaceIsTypeCheck(), InterfaceIsTypeCheckTests.TEST1, "Interfaces should describe a type and hence have methods");
```

Comment:
FlashDevelop doesn't particularly like it either, but I think it still helps readability because of the shorter lines.